accord-core/src/main/java/accord/impl/MajorityDepsFetcher.java

```diff
@@ -208,6 +209,8 @@ private void start(Range slice, int nextIndex)
                                             schedule();
                                         }
                                     }
+                                    if (notifyWaiting)
```

Comment:
> 3) MajorityDepsFetcher would have a deadlock as it triggers waiting notifications while holding the lock, and the waiting callers then access more locks, such as the config service lock

`trySuccess` will trigger all callbacks, which then leads to a deadlock as we never release this lock

Reply:
this code is all defunct now in trunk